Dockerfile

```diff
@@ -1,26 +1,27 @@
 FROM ubuntu:14.04
-MAINTAINER Bhavya bchandra@hsr.ch
+MAINTAINER Geometalab geometalab@hsr.ch
```

Comment:
When combining a name and an email address, put the email address into pointy brackets `<...>`. (Just the email address would suffice here, too. Then no brackets are required AFAIK.)

Though, the `MAINTAINER` DOCKERFILE command [has been deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). Dunno whether we should set the suggested label instead of just don't declare the image author at all.